WordPress/Classes/ViewRelated/Post/PostSettingsViewController.m

```diff
@@ -532,7 +531,8 @@ - (CGFloat)tableView:(UITableView *)tableView heightForRowAtIndexPath:(NSIndexPa
     }
 
     if (sectionId == PostSettingsSectionMeta) {
-        if (indexPath.row == RowIndexForPassword) {
+        NSInteger row = [[self.postMetaSectionRows objectAtIndex:indexPath.row] integerValue];
```

Comment:
This isn't the prettiest, so if you think we would benefit stylistically by breaking it out into a function that this and `configureMetaPostMetaCellForIndexPath` would call, I'll be glad to.

Reply:
I think it's ok, I don't have any strong opinions about this one.